packages/serverpod/lib/src/database/adapters/postgres/value_encoder.dart

```diff
@@ -31,7 +31,7 @@ class ValueEncoder extends PostgresTextEncoder {
       // encoded to base64. Best would be to find a better way to detect when we
       // are trying to store a ByteData.
       return input;
-    } else if (input is SerializableEntity && input is Enum) {
+    } else if (input is SerializableModel && input is Enum) {
```

Comment:
Question: Is there ever a case where we encode the data in the protocol INTO a `SerializableEntity`? So that we won't be able to decode it properly after this change? Say for example when `encodeWithType(...)` is called?

Reply:
We never do this, so i dont think this is never gonna be an issue!